src/theme/ThemedComponents/Badge/LabelBadge.js

```diff
@@ -0,0 +1,137 @@
+import React from "react";
+import Chip from "@material-ui/core/Chip";
+import theme from "../../muiTheme";
+import { Typography } from "@material-ui/core";
+import PropTypes from "prop-types";
+
+export const LabelBadge = ({label, variant = "default", style, ...rest}) => {
```

Comment:
instead of only having a "label" prop, let's also allow the user to submit using a children. Right now to add a label to the LabelBadge component you have to do `<LabelBadge label="something"/>` but it's more natural for developers to use it like `<LabelBadge> something </LabelBadge>` so to do this we need to use react component's children (anything in between an open and close tags). ask @pswk1 if you have questions about this

Reply:
LabelBadges now take label as a child